WeakAuras/RegionTypes/AuraBar.lua

```diff
@@ -893,6 +936,12 @@ local function modify(parent, region, data)
   bar.spark.sparkRotation = data.sparkRotation;
   bar.spark.sparkMirror = data.sparkMirror;
 
+    -- Gradient
+  region.bar.gradient = data.gradient
+  region.bar.gradientDirection = data.gradientDirection
+  region.bar.gradientColor = data.gradientColor
+  region.bar.updateGradient = data.gradient and data.orientation:match(data.gradientDirection) 
```

Comment:
This doesn't catch orientation changes via Conditions.

https://i.imgur.com/TbkNYho.gifv

Reply:
Good catch! Thanks, I'll look into this.